src/middlewared/middlewared/etc_files/local/sudoers

```diff
@@ -0,0 +1,22 @@
+<%
+	import os
+
+	users = middleware.call_sync('user.query', [["sudo", "=", True]])
+	groups = middleware.call_sync('group.query', [["sudo", "=", True]])
+
+	if os.path.isdir('/conf/base/usr__local__etc'):
+		base_file = '/conf/base/usr__local__etc/sudoers'
+	else:
+		base_file = '/conf/base/etc/local/sudoers'
```

Comment:
sudoers is a local customization, this should be removed and contents of the file placed here.